micro-ui/web/micro-ui-internals/packages/svg-components/src/svg/ShpIcon.js

```
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export const ShpIcon = ({className, height="72", width="69", fill="none"}) => (
+
+    <svg className={className} width={width} height={height} viewBox="0 0 69 72" fill={fill} xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink">
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)